app/javascript/flavours/polyam/features/compose/components/emoji_picker_dropdown.jsx

```diff
@@ -277,9 +279,12 @@ class EmojiPickerMenuImpl extends PureComponent {
     return (
       <div className={classNames('emoji-picker-dropdown__menu', { selecting: modifierOpen })} style={style} ref={this.setRef}>
         <EmojiPicker
+          data={data}
           perLine={8}
           emojiSize={22}
           sheetSize={32}
+          sheetColumns={61}
+          sheetRows={61}
```

Comment:
Instead of manually having to update these properties,  they could be added to data, but it doesn't make any functional difference.
Again, wrapping the component would allow to change the defaults and not make this necessary.